### PR TITLE
Show outage banner on Open Library for mobile

### PR DESCRIPTION
**Pull request by mekarpeles**
Followup to #5313 

### Stakeholders

@cdrini

### CODE REVIEW

**Comment by jamesachamp** [comment]
This seems to look alright when tested locally:

local-responsive-banner

File: static/css/components/page-banner.less [hunk 1]
```diff
@@ -62,5 +62,6 @@
     padding: 10px;
     line-height: 25px;
     text-align: center;
+    display: block;
```
Comment (cdrini) [inline static/css/components/page-banner.less:65]: ^^ Further up in this file, there's `display:none` on the main `.page-banner` element; let's knock that off, and the corresponding media query that displays the banner on small screens.

**Review by cdrini — CHANGES_REQUESTED** [review]

**Comment by mekarpeles** [comment]
Was this one ever patch deployed?

**Comment by mekarpeles** [comment]
I see that it was here:
https://github.com/internetarchive/openlibrary/pull/5313/files#r651496174
Thank you!

**Comment by cdrini** [comment]
All done and a bottle of rum!

**Comment by cdrini** [comment]
Ooops, closed the wrong one :P

**Comment by jamesachamp** [comment]
Resolved with inline CSS in #5313

**Pull request closed without merging**